sdk/lib/atl/atlwin.h

```diff
@@ -575,6 +575,17 @@ class CWindow
         return ::GetDlgItemText(m_hWnd, nID, lpStr, nMaxCount);
     }
 
+#ifdef __ATLSTR_H__
+    UINT GetDlgItemText(int nID, CSimpleString& string)
+    {
+        HWND item = GetDlgItem(nID);
+        int len = ::GetWindowTextLength(item);
+        len = GetDlgItemText(nID, string.GetBuffer(len+1), len+1);
+        string.ReleaseBuffer(len+1);
```

Comment:
`ReleaseBuffer` expects the length without the null terminator, so just `len` is good.